plugins/chat/app/models/chat/message.rb

```diff
@@ -76,6 +76,9 @@ class Message < ActiveRecord::Base
     def self.polymorphic_class_mapping = { "ChatMessage" => Chat::Message }
 
     def validate_message(has_uploads:)
+      self.message =
```

Comment:
I kinda wanted to avoid this... as we change data in a function called "validate" it's somewhat unexpected. But I also know it's complicated to do differently atm :/

Reply:
Yeah I really don't like it much either, this whole validation should be inside an actual AR validation because then we can do stuff like `before_validation`. But yeah kinda too complex ATM